CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 5.0.0-wip
+
+- `recommended`:
+  - added `unnecessary_library_name` (https://github.com/dart-lang/lints/issues/181)
```

Comment:
Consider linking the lint name to https://dart.dev/lints/unnecessary_library_name so readers of the changelog can learn more :)